data/sql/updates/pending_db_world/rev_1643656464415474000.sql

```diff
@@ -0,0 +1,82 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643656464415474000');
+
+-- All Blackwing Technician's spawns
+DELETE FROM `creature` WHERE `id1` IN (13996);
```

Comment:
Delete GUID as well. That is the Primary key :D

Reply:
Oh man, that's gonna be a nasty delete. So many Guids

Reply:
Yea, but mandatory as GUID is the key field